Crashlytics/Crashlytics/Models/Record/FIRCLSReportAdapter.m

```diff
@@ -155,19 +155,21 @@ - (google_crashlytics_FilesPayload)protoFilesPayload {
   google_crashlytics_FilesPayload_File *files =
       malloc(sizeof(google_crashlytics_FilesPayload_File) * clsRecords.count);
 
-  for (NSUInteger i = 0; i < clsRecords.count; i++) {
-    google_crashlytics_FilesPayload_File file = google_crashlytics_FilesPayload_File_init_default;
-    file.filename = FIRCLSEncodeString(clsRecords[i].lastPathComponent);
-
-    NSError *error;
-    file.contents = FIRCLSEncodeData([NSData dataWithContentsOfFile:clsRecords[i]
-                                                            options:0
-                                                              error:&error]);
-    if (error) {
-      FIRCLSErrorLog(@"Failed to read from %@ with error: %@", clsRecords[i], error);
+  if (files != NULL) {
```

Comment:
More of a style thing, but I like to keep the happy path code in the least-indent, and use if statements to check for issues. Can we change this to:

if (files == NULL) {
  return apple_payload;
}

Reply:
Done.